samples/matter/common/src/persistent_storage/persistent_storage_shell.cpp

```diff
@@ -6,21 +6,39 @@
 
 #include "persistent_storage_shell.h"
 
+#ifdef CONFIG_NVS
 #include <zephyr/fs/nvs.h>
+#elif CONFIG_ZMS
+#include <zephyr/fs/zms.h>
+#endif
 #include <zephyr/settings/settings.h>
 #include <zephyr/shell/shell.h>
 
 using namespace Nrf;
 
 namespace
 {
-nvs_fs *sNvsStorage = nullptr;
+#ifdef CONFIG_NVS
+nvs_fs *sStorage = nullptr;
+#elif CONFIG_ZMS
+zms_fs *sStorage = nullptr;
+#endif
```

Comment:
```suggestion
#if defined(CONFIG_NVS)
...
#elif defined(CONFIG_ZMS)
...
#endif
```
and apply throughout